internal/action/actions.go

```diff
@@ -234,6 +234,22 @@ func (h *BufPane) CursorRight() bool {
 	return true
 }
 
+// SubWordRight moves the cursor one sub-word to the right
+func (h *BufPane) SubWordRight() bool {
+	h.Cursor.Deselect(false)
+	h.Cursor.SubWordRight()
+	h.Relocate()
+	return true
+}
+
+// SubWordLeft moves the cursor one sub-word to the left
+func (h *BufPane) SubWordLeft() bool {
+	h.Cursor.Deselect(true)
+	h.Cursor.SubWordLeft()
+	h.Relocate()
+	return true
+}
+
```

Comment:
nit: Move `SubWordRight()` and `SubWordLeft()` after `WordRight()` and `WordLeft()`?